modules/repository-azure/build.gradle

```diff
@@ -17,32 +17,47 @@ esplugin {
 }
 
 versions << [
-  'azure': '12.20.1',
-  'azureCommon': '12.19.1',
-  'azureCore': '1.34.0',
-  'azureCoreHttpNetty': '1.12.7',
-  'azureJackson': '2.13.4',
-  'azureJacksonDatabind': '2.13.4.2',
-  'azureAvro': '12.5.3',
-
-  'jakartaActivation': '1.2.1',
-  'jakartaXMLBind': '2.3.2',
+  //see https://repo1.maven.org/maven2/com/azure/azure-storage-blob/ for transitive dependencies
+  'azure': '12.26.1', // <-- when upgrading versions start start here and manually upgrade the transitive dependencies
```

Comment:
I upgraded this version to the latest non-beta release and then manually found all correct transitive versions. I updated the comments here with where I pulled the version numbers from.